stories/core/HorizontalTabs.stories.tsx

```diff
@@ -49,7 +49,7 @@ BorderlessTabs.args = {
 export const PillTabs = Template.bind({});
 PillTabs.args = {
   labels: [...new Array(3)].map((_, index) => (
-    <SvgStar key={index} aria-hidden />
+    <SvgStar key={index} aria-hidden className='iui-tab-icon' />
```

Comment:
This should change with another PR, right?

Reply:
Yup